packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.service.js

```diff
@@ -177,9 +178,10 @@ export default class SofpthoneService {
             'Content-type': 'multipart/form-data',
           },
         })
-        .then(() => putUrl);
+        .then(() => globalGetUrl);
```

Comment:
Is it expected ? `globalGetUrl` is null here

Reply:
nope is not because its value changes